apps/docs/src/content/docs/libraries/auth.mdx

```diff
@@ -0,0 +1,115 @@
+---
+title: auth
+description: Authentication functionality provider.
+---
+
+## 📦 Overview
+
+### What it does
+
+This library is the authentication layer using [Lucia Auth](https://lucia-auth.com/).
```

Comment:
TODO: Replace NextAuth in [Tools Overview](https://docs.cuhacking.ca/docs/tools-overview#%EF%B8%8F-back-end--middleware)